src/gameComponents/DiceSymbol.java

```diff
@@ -0,0 +1,5 @@
+package gameComponents;
+
+public enum DiceSymbol {
+    CRIT, HIT, ACCURACY
```

Comment:
blanks?

Reply:
Intentionally left out.  They aren't a symbol.

This is to count the occurances of a symbol to account for double icon faces.  Might have a side of yagni given the values stored in the enums.